### DISCUSSION TITLE
I am inside the campus but Github is detecting i am not inside campus.

### DISCUSSION THREAD

**Discussion by carzey-sol** [community/community#145451] [GitHub Education]
### Hi everyone,

I am trying to signup for a student developer pack but i am getting a issue even if i am inside the campus, the system is detecting that i an not inside the campus.
![image](https://github.com/user-attachments/assets/3c5581fd-a0e6-4364-bb01-f0ddb34b62d5)

**Comment by davevad93** [comment]
Hi @carzey-sol ,

It's possible that GitHub has incorrect information about your school, contact support by opening a ticket [here](https://support.github.com/contact/education), select `My selected school might have incorrect or incomplete information e.g. domains or campus location` and explain your issue.
